docs/examples/soroban/restore_footprint.md

```diff
@@ -0,0 +1,94 @@
+# Restore Footprint
+The `RestoreFootprint` operation is used to restore a contract data entry's.The restored entry will have its expiration ledger bumped to the minimums the network allows for newly created entries, which is 4096 + current ledger for persistent entries, and 16 + current ledger for temporary entries.
```

Comment:
- Add link in "mininums" - https://github.com/stellar/stellar-core/blob/2109a168a895349f87b502ae3d182380b378fa47/src/ledger/NetworkConfig.h#L77-L78
- Add a space before "The restored entry..."